app/javascript/components/workflows/student/exams/show/components/TooltipButton.tsx

```diff
@@ -21,22 +22,26 @@ const TooltipButton: React.FC<TooltipButtonProps> = (props) => {
     variant = 'primary',
     placement = 'bottom',
     children,
+    className,
   } = props;
   return (
     <Tooltip
       showTooltip={disabled}
       message={disabledMessage}
       placement={placement}
-      className={`d-inline-block ${disabled && 'cursor-help'}`}
     >
-      <Button
-        variant={variant}
-        onClick={onClick}
-        disabled={disabled}
-        className={disabled && 'pointer-events-none'}
+      <span
+        className={`d-inline-block ${disabled && 'cursor-help'}`}
```

Comment:
I know we deliberately changed this to `cursor-help` at some point, and I think that's useful for things like the cloud-save-status button.  But for disabled buttons like Delete, it seems to me that the cursor-not-allowed icon would be more intuitive -- this help icon just looked weirdly wrong to me.